spec/policies/address_policy_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe AddressPolicy do
+  let(:user) { User.new }
+
+  subject { described_class }
+
+  permissions :index? do
+    it "is permited for anyone" do
+      expect(subject).to permit(nil, User)
+    end
+  end
+end
```

Comment:
Nothing to document. I removed it, since it isn't being used. It's just something I missed on first pass.